resources/dicts/events/injury/general.json

```diff
@@ -7214,5 +7214,187 @@
+        "location": ["any"],
+        "season": ["any"],
+        "sub_type": ["war"],
+        "weight": 10,
+        "event_text": "Yet another skirmish with o_c_n takes place, though this one is interrupted by a dog. On instinct, the skirmishers unite to drive off the slavering beast. After, the patrols tend the injured - m_c among them - in a moment that makes it hard to remember they're on opposite sides of a war.",
+        "m_c": {
+            "status": ["warrior"]
+        },
+        "injury": [
+            {
+                "cats": [
+                    "m_c"
+                ],
+                "injuries": [
+                    "small_bite_injury"
+                ]
+            }
+        ],
+        "history": [
+            {
+                "cats": [
+                    "m_c"
+                ],
+                "scar": "m_c was scarred after a dog attacked during a skirmish.",
+                "reg_death": "m_c died from {PRONOUN/m_c/poss} wounds after a dog attacked during a skirmish.",
+                "lead_death": "died from wounds after a dog attacked during a skirmish"
+            }
+        ],
+        "other_clan": {
+            "current_rep": ["hostile"],
+            "changed": 1
+        }
+    },
+    {
+        "event_id": "gen_misc_war_medcat_assist1",
+        "location": ["any"],
+        "season": ["any"],
+        "sub_type": ["war"],
+        "weight": 10,
+        "event_text": "Things are looking bleak for m_c, struggling with severe injuries in the wake of a skirmish with o_c_n. Unexpectedly, the enemy Clan's medicine cat emerges from the shadows, tending to the injuries until c_n's own medicine cats can take over.",
+        "m_c": {
+            "status": ["warrior"]
+        },
+        "injury": [
+            {
+                "cats": [
+                    "m_c"
+                ],
+                "injuries": [
+                    "claw-wound",
+                    "torn pelt"
+                ]
+            }
+        ],
+        "history": [
+            {
+                "cats": [
+                    "m_c"
+                ],
+                "scar": "m_c was scarred after a skirmish with a warring Clan.",
+                "reg_death": "m_c died from {PRONOUN/m_c/poss} wounds after a skirmish with a warring Clan.",
+                "lead_death": "died from wounds after a skirmish with a warring Clan"
+            }
+        ],
+        "other_clan": {
+            "current_rep": ["hostile"],
+            "changed": 1
+        }
+    },
+    {
+        "event_id": "gen_misc_war_medcat_assist2",
+        "location": ["any"],
+        "season": ["any"],
+        "sub_type": ["war"],
+        "weight": 20,
+        "event_text": "Having heard that o_c_n was running low on vital emergency herbs, m_c gathers up some excess stock and makes the trek. However, with a mouth full of leaves, m_c takes a claw to the face before {PRONOUN/m_c/subject} can explain {PRONOUN/m_c/poss} intentions.",
```

Comment:
while not totally necessary, it might be good to have this actually be based upon the clan's current herb supply.  i could see a potential scenario where the clan actually has no (or very low) herb and this becomes a contradictory event.  you could set it up to trigger on excess of a random herb, then "destroy" a small amount of that herb.  you could even then create some other similar events that are triggered by other herb supply states, such as one where the clan's supply actually *is* low, but they *still* give the herbs to their enemy and thus gain a larger positive relation increase.